locale/it/LC_MESSAGES/django.po

```diff
@@ -1006,24 +1006,24 @@ msgstr "Conto"
 #: meta/templates/meta/heroes.html:12
 #, python-format
 msgid "Unpicked Heroes (%(unpicked_heroes_count)s of %(heroes_count)s)"
-msgstr ""
+msgstr "Eroi non scelti (%(unpicked_heroes_count) su %(heroes_count)"
```

Comment:
I think this should be `%(unpicked_heroes_count)s su %(heroes_count)s`, could you please update?